python_utils.py

```diff
@@ -429,3 +429,29 @@ def yaml_from_dict(dictionary, width=80):
     """
     dictionary = _recursively_convert_to_str(dictionary)
     return yaml.safe_dump(dictionary, default_flow_style=False, width=width)
+
+
+def compress_to_zlib(data):
```

Comment:
python_utils.py should only be used for stuff that's different in py2 and py3. You might want just utils.py instead.

Reply:
I see. Thanks. Will move this.

Reply:
Moved to utils